components/table/action.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+
+@observer
+class Action extends React.Component {
```

Comment:
Why do you need empty observer wrapper?

Would not having observer component in place solve your problem?

```js
const TableButton = observer(ExportButton)
```

Reply:
It was just copy-paste of the Column component. The observer is no needed there.